tests/PHPStan/Analyser/AnalyserIntegrationTest.php

```diff
@@ -278,9 +278,10 @@ public function testBug3379(): void
 			$this->markTestSkipped('Test requires static reflection');
 		}
 		$errors = $this->runAnalyse(__DIR__ . '/data/bug-3379.php');
-		$this->assertCount(2, $errors);
+		$this->assertCount(3, $errors);
 		$this->assertSame('Constant SOME_UNKNOWN_CONST not found.', $errors[0]->getMessage());
 		$this->assertSame('Reflection error: Could not locate constant "SOME_UNKNOWN_CONST" while evaluating expression in Bug3379\Foo at line 8', $errors[1]->getMessage());
+		$this->assertSame('Reflection error: Could not locate constant "SOME_UNKNOWN_CONST" while evaluating expression in Bug3379\Foo at line 8', $errors[2]->getMessage());
```

Comment:
atm I have no clue, why this additional error is coming up

Reply:
ahh just realized this error is also present on the latest master branch:
https://github.com/phpstan/phpstan-src/runs/3374000058

therefore it should not be related to the changes from this PR.

Reply:
Removed the hack. This should be good to go